packages/ui/src/components/Checkbox.tsx

```diff
@@ -1 +1,44 @@
-export { type CheckboxProps, Checkbox } from 'theme-ui';
+import * as React from 'react';
+import {
+  Checkbox as CheckBoxUi,
+  CheckboxProps as CheckboxPropsUi,
+} from 'theme-ui';
+
+export interface CheckboxProps extends CheckboxPropsUi {
+  indeterminate?: boolean;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  className?: string;
+}
+
+export const Checkbox: React.FC<CheckboxProps> = ({
```

Comment:
We don't use React.FC https://github.com/ttoss/ttoss/pull/497

Reply:
In my test this function corrige the error on ref.current, but i corrige with my interface